src/index.js

```diff
@@ -178,6 +183,23 @@ client.on("messageReactionAdd", async (messageReaction, user) => {
 });
 /* --------------------------------------------------- */
 
+function isDm(message) {
+	return message.channel.type === 'dm';
+}
+
+function handleDmReactionAdd(emoji, message) {
+	if (emoji.name === '❌' && isMessageFromBot(message)) {
+		message.delete().catch(function (error) {
+			console.log(error);
+		});
+		return;
+	}
+}
```

Comment:
What are your thoughts on having this as a separate module? Perhaps in `src/scripts/users/dm/` and placing `bookmarks.js` there as well?
I think it'll be good to move away from having a large index.js file as much as possible.

As we move towards a more logically structured folder system I'm hoping we can also clean up our require statements down the road.

Reply:
Yeah, that sounds good! DM operations definitely seem distinct to normal channel messages even though they're fired off in the same message listener so splitting them off into their own module would make sense

Unrelated to this, I think our `scripts` folder could be renamed, or removed and have its contents moved around into other folders, it's a bit of an odd name since everything we have is a script so it looks more like a "miscellaneous" folder

Reply:
Yeah, I definitely agree. Sounds like a plan! I'd especially like to have the `utilities.js` moved to the `utils` folder and the channels files grouped as well next.